Modules/vc-module-export/VirtoCommerce.ExportModule.Core/Model/IExportable.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+
+namespace VirtoCommerce.ExportModule.Core.Model
+{
+    /// <summary>
+    /// Interface to implement exportаble entities.
+    /// Common (default) set of properties for universal viewing.
+    /// </summary>
+    public interface IExportable : ICloneable
```

Comment:
`IExportable` interface shouldn't contains fields are intended for presentation in UI it should be a separate interface `IExportViewable`  or some like that.
`IExportable` must be or empty or be derived from `IEntity` and `ICloneable` interfaces. And all export services must operate `IExportable` interfaces instead of object